script.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+/**
+ * Load the TypeScript compiler, then load the TypeScript gulpfile which simply loads all
```

Comment:
not sure if this file is meant to be in this PR, but either way the comment is copy/pasta'd from somewhere else 😄.

Reply:
haha, yeah. will remove :P